config/locales/client.en.yml

```diff
@@ -80,9 +80,12 @@ en:
 
       reset:
         title: "Disable encrypted messages"
-        instructions: "Are you sure you want to disable encrypted messages and reset your encryption key? This operation deletes your encryption key forever. You will not be able to access any of your current encrypted private messages unless you have the exported key pair. However, you can enable encrypted messages by generating a new key pair at any later time."
+        instructions: "Are you sure you want to disable encrypted messages and reset your encryption key? This operation deletes your encryption key forever and you will not be able to access any of your encrypted private messages anymore."
+        instructions_2:
```

Comment:
I'm not a big fan of the subscript `_2` in the key here. Could we use something like `warning`?